air/src/proof/context.rs

```diff
@@ -109,36 +90,24 @@ impl<E: StarkField> ToElements<E> for Context {
     /// Converts this [Context] into a vector of field elements.
     ///
     /// The elements are laid out as follows:
-    /// - trace layout info [1 or more elements].
+    /// - trace info [1 or more elements].
```

Comment:
Trace info takes "2 or more elements", right? (because it will have main segment info, and also now trace length).